psi4/driver/driver.py

```diff
@@ -188,6 +189,11 @@ def _process_displacement(derivfunc, method, molecule, displacement, n, ndisp, *
     derivative, wfn = derivfunc(method, return_wfn=True, molecule=clone, **kwargs)
     displacement["energy"] = core.variable('CURRENT ENERGY')
 
+    # Getting the dipole moment and saving it a numpy array.
+    displacement["dipole"] = np.array([core.variable('CURRENT DIPOLE X'),
+                                      core.variable('CURRENT DIPOLE Y'),
+                                      core.variable('CURRENT DIPOLE Z')])/constants.dipmom_au2debye
```

Comment:
Please YAPF this file when you have a moment.

Reply:
Should I also YAPF the driver_findif.py, which is the other file I modified?  Also should I YAPF only portions I modifed or the whole file? Because YAPF suggest changes to code I didn't write.

Reply:
if it's a wall of changes, no; if its under a dozen, yes, imo.